apps/web/src/features/layout/components/Header/header.module.css

```
@@ -897,3 +901,154 @@
+  color: #fffbeb !important;
+  text-shadow: 0 0 12px rgba(249, 115, 22, 0.7) !important;
+  box-shadow: 0 0 14px rgba(245, 158, 11, 0.2) !important;
+}
+
+:global([data-theme="phoenix"][data-mode="light"]) .navLink:hover {
+  background-color: rgba(234, 88, 12, 0.12) !important;
+  color: #7c2d12 !important;
+  text-shadow: 0 0 8px rgba(234, 88, 12, 0.4) !important;
+}
+
+:global([data-theme="phoenix"]) .navLink:focus-visible {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)